.editorconfig

```diff
@@ -0,0 +1,26 @@
+# http://editorconfig.org/
+
+root = true
+
+[*]
+end_of_line = lf
+insert_final_newline = true
+charset = utf-8
+trim_trailing_whitespace = true
+
+[*.go]
+indent_style = tab
+
+[*.json]
+indent_size = 2
+
+[Makefile,*.mk]
+indent_style = tab
+
+[*.md]
+indent_style = space
+indent_size = 4
+
+[.golangci.yml]
```

Comment:
Maybe we merge this with the `*.json` rule?

Reply:
With the change to `indent_size = 4` above, I think we can leave it separate now?

Reply:
0/5, it can stay